### DISCUSSION TITLE
kex_exchange_identification: Connection closed by remote host

### DISCUSSION THREAD

**Discussion by Abhi0725** [spatie/laravel-server-monitor#126] [Q&A]
Sometimes getting the following error:

`Failed to run: kex_exchange_identification: Connection closed by remote host Connection closed by aa.bbb.ccc.d port 22`

This doesn't happen every time and also, auto fixes within a minute. I know this can happen because of various reasons but anything anyone here faced it and solved?

**Comment by neochief** [comment]
I also get this all the time randomly.
